esmvaltool/cmorizers/obs/cmorize_obs_merra2.py

```diff
@@ -0,0 +1,158 @@
+"""ESMValTool CMORizer for MERRA2 data.
+
+Tier
+    Tier 3: restricted datasets (i.e., dataset which requires a registration
+ to be retrieved or provided upon request to the respective contact or PI).
+
+Source
+    https://goldsmr4.gesdisc.eosdis.nasa.gov/data/MERRA2_MONTHLY/
```

Comment:
It would be better to get this from `config-user.yml` (e.g., `RAWOBS`) or at least to have it as a command line argument.